Linux ARM64 support

<!--
IF SUFFICIENT INFORMATION IS NOT PROVIDED VIA THE FOLLOWING TEMPLATE THE ISSUE MIGHT BE CLOSED WITHOUT FURTHER CONSIDERATION OR INVESTIGATION
-->
> Please provide us with the following information:
> ---------------------------------------------------------------

Please provide a buildArm64Linux script as the ARM32 does not run on ARM64 platforms like NVidia Jetson Nano.

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [X] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>
Building Linux Arm32 sample ...
/usr/bin/ld: skipping incompatible ./lib/arm32/libMicrosoft.CognitiveServices.Speech.core.so when searching for -lMicrosoft.CognitiveServices.Speech.core
/usr/bin/ld: cannot find -lMicrosoft.CognitiveServices.Speech.core
collect2: error: ld returned 1 exit status
Cleaning up downloaded files
Done. To start the demo execute:

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
Ubuntu 18-04
Linux jetson 4.9.140-tegra #1 SMP PREEMPT Wed Apr 8 18:10:49 PDT 2020 aarch64 aarch64 aarch64 GNU/Linux

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

--- a/clients/cpp-console/scripts/linux/buildArmLinux.sh
+++ b/clients/cpp-console/scripts/linux/buildArmLinux.sh
@@ -1,0 +1,83 @@
+#!/bin/bash
+clear
+cd ../..
+if [ ! -d out ]; then
+    mkdir out # only create directory if does not exist
+fi
+if [ ! -d SDK ]; then
+    mkdir SDK # only create directory if does not exist
+fi
+
+# Determine if the Linux is ARM32 or ARM64
+ARCH=$(uname -m)
+if [[ $ARCH =~ "v7" ]] # ARM32 OS in use
+then
+echo "Running on ARM32 Linux"
+    #echo "Running on ARM32 Linux, creating lib/lib link to lib/arm32"
+LIBLINK="arm32"
+    #ln -s arm32 lib/lib
+else
+    #echo "Running on ARM64 Linux, creating lib/lib link to lib/arm64"
+    #ln -s arm64 lib/lib
+echo "Running on ARM64 Linux"
+LIBLINK="arm64"
+fi
+
+if [ ! -f ./lib/lib/libMicrosoft.CognitiveServices.Speech.core.so ]; then
+echo "Cleaning up libs and include directories that we will overwrite"
+rm -R ./lib/*
+rm -R ./include/c_api
+rm -R ./include/cxx_api
+
+echo "Downloading Speech SDK binaries"
+wget -c https://aka.ms/csspeech/linuxbinary -O - | tar -xz -C ./SDK
+
+echo "Copying SDK binaries to lib folder and headers to include"
+cp -Rf ./SDK/SpeechSDK*/lib .
+cp -Rf ./SDK/SpeechSDK*/include .
+
+echo "Creating lib/lib link to lib/$LIBLINK"
+ln -s $LIBLINK lib/lib
+
+else
+echo "Speech SDK lib found. Skipping download."
+fi
+
+echo "Building Linux Arm sample ..."
+if g++ -Wno-psabi \
+src/common/Main.cpp \
+src/linux/LinuxAudioPlayer.cpp \
+src/common/AudioPlayerEntry.cpp \
+src/common/AgentConfiguration.cpp \
+src/common/DeviceStatusIndicators.cpp \
+src/common/DialogManager.cpp \
+-o ./out/sample.exe \
+-std=c++14 \
+-D LINUX \
+-D MAS \
+-L./lib/lib \
+-I./include/cxx_api \
+-I./include/c_api \
+-I./include \
+-pthread \
+-lstdc++fs \
+-lasound \
+-lMicrosoft.CognitiveServices.Speech.core; 
+then
+error=0;
+else
+error=1;
+fi
+
+cp ./scripts/run.sh ./out
+chmod +x ./out/run.sh
+
+echo Cleaning up downloaded files
+rm -R ./SDK
+
+echo Done. To start the demo execute:
+echo cd ../../out
+echo export LD_LIBRARY_PATH="../lib/lib"
+echo ./sample.exe [path_to_configFile]
+
+exit $error

